README.md

```diff
@@ -1943,6 +1943,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sudhansu](https://github.com/the-lazy-learner)
   - [Sudochain](https://github.com/sudochain)
   - [Sugam50](https://github.com/Sugam50)
+  - [SuhasA](https://github.com/suhasamaresh)  
```

Comment:
Please add your name in line 1949, after Suhail Khan.
Let us know if you need any help with that. 🙂

Reply:
ohh that's right 👀

Reply:
yeah i updated!